metrics/src/logsToMetrics/Function.cs

```diff
@@ -22,7 +23,8 @@ public class Function
             "/aws/lambda/net-7-base-x86-64", // TODO: Add ARM for .NET 7+ when the supported build image is released
             "/aws/lambda/net-7-native-x86-64",
             "/aws/lambda/net-8-base-x86-64",
-            "/aws/lambda/net-8-native-x86-64"
+            "/aws/lambda/net-8-native-x86-64",
+            "/aws/lambda/net-6-customRuntime-x86-64"
```

Comment:
nit: can you sort these alphabetically so that net-6-customRuntime is next to net-6-base

Reply:
done